readme.md

```diff
@@ -47,13 +47,15 @@ const {app, BrowserWindow, ipcMain} = require('electron');
 const {download} = require('electron-dl');
 
 ipcMain.on('download-btn', (e, args) => {
-	download(BrowserWindow.getFocusedWindow(), args.url);
+	download(BrowserWindow.getFocusedWindow(), args.url)
```

Comment:
I amended the existing example instead of adding a completely separate one showing Promise use.